tests/phpunit/class-wc-stripe-connect-test.php

```
@@ -0,0 +1,105 @@
+				'type'        => 'app',
+				'expected_oc' => '',
+				'expected_ap' => '',
+			],
+			'no marker + connect type leaves both unset'                       => [
+				'marker_set'  => false,
+				'type'        => 'connect',
+				'expected_oc' => '',
+				'expected_ap' => '',
+			],
+		];
+	}
```

PHPMD (2.15.0)

[error] The method provide_save_stripe_keys_defaults_ocs_for_new_installs is not named in camelCase. (undefined)

(CamelCaseMethodName)